src/main/java/br/com/zup/library/exemplar/Exemplar.java

```diff
@@ -0,0 +1,44 @@
+package br.com.zup.library.exemplar;
+
+import br.com.zup.library.livro.Livro;
+
+import javax.persistence.*;
+
+import static javax.persistence.EnumType.STRING;
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+public class Exemplar {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    private Long id;
+    @Enumerated(STRING)
+    private Circulacao circulacao;
+    @ManyToOne
+    private Livro livro;
+
+    public Exemplar(String circulacao, Livro livro) {
```

Comment:
Aqui você está abrindo mão do poder da tipagem para facilitar uso do construtor nessa atividade. Você acha que vale a pena essa troca?

Quem for instanciar a entidade `Exemplar` sabe quais as possibilidades de circulação? Sabe a formatação?

Reply:
Corrigido